<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <title>Dakota Hawkins - Published Work</title>
  <meta name="viewport" content="width=device-width">

  <!-- syntax highlighting CSS -->
  <link rel="stylesheet" href="/css/syntax.css">

  <!-- Custom CSS -->
  <link rel="stylesheet" href="/css/main.css">

  <!-- Responsive CSS -->
  <link rel="stylesheet" href="/css/responsive.css">

  <!-- Google Fonts -->
  <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Raleway:300,400,700' rel='stylesheet' type='text/css'>
</head>

<body>
  <div class="site">
    <div class="sidebar" id="sidebar">
      <div class="header">
        <h1 class="title"><a href="/"><img id="logo" src="/images/logo.jpg"></a></h1>
        <span class="tagline">Dakota Hawkins</span>
      </div>
      <div class="posts">
        <ul class="posts-list">
          
          <li class="post-link">
            <a class="post-title" href="/posts/2023/07/12/about.html">
              <!-- <span class="post-date">12 Jul 2023</span> -->
              About Me
            </a>
          </li>
          
          <li class="post-link">
            <a class="post-title" href="/posts/2023/07/12/thesis.html">
              <!-- <span class="post-date">12 Jul 2023</span> -->
              Thesis
            </a>
          </li>
          
          <li class="post-link">
            <a class="post-title" href="/posts/2023/07/12/projects.html">
              <!-- <span class="post-date">12 Jul 2023</span> -->
              Published Work
            </a>
          </li>
          
        </ul>
      </div>
      <div class="footer">
        <span id="footer-links">
          <a href="" class="footer-link">About</a> <span class="separator">&bull;</span>
          <a href="https://www.linkedin.com/in/dakota-hawkins-a901889a/" class="footer-link">LinkedIn</a> <span
            class="separator">&bull;</span>
          <a href="https://github.com/dakota-hawkins" class="footer-link">GitHub</a>
        </span>
      </div>
    </div>

    <div class="content" id="home">
  <div id="sidebar-button">
    <img src="/images/sidebar-button.png">
  </div>
  <div id="post-info">
    <div id="cover-photo-container">
      <img id="cover-photo" src="/images/solve_smol.png">
    </div>
    <div id="info-container">
      <h1 id="title">Published Work</h1>
    </div>
  </div>
  <br />
  <div class="post">
    <h2 id="icat-a-novel-algorithm-to-robustly-identify-cell-states-following-perturbations-in-single-cell-transcriptomes">ICAT: a novel algorithm to robustly identify cell states following perturbations in single-cell transcriptomes</h2>

<p><a href="https://academic.oup.com/bioinformatics/article/39/5/btad278/7136641">Bioinformatics, 2023</a></p>

<p><strong>Abstract:</strong></p>

<p><strong>Motivation</strong>
The detection of distinct cellular identities is central to the analysis of single-cell RNA sequencing (scRNA-seq) experiments. However, in perturbation experiments, current methods typically fail to correctly match cell states between conditions or erroneously remove population substructure. Here, we present the novel, unsupervised algorithm Identify Cell states Across Treatments (ICAT) that employs self-supervised feature weighting and control-guided clustering to accurately resolve cell states across heterogeneous conditions.</p>

<p><strong>Results</strong>
Using simulated and real datasets, we show ICAT is superior in identifying and resolving cell states compared with current integration workflows. While requiring no a priori knowledge of extant cell states or discriminatory marker genes, ICAT is robust to low signal strength, high perturbation severity, and disparate cell type proportions. We empirically validate ICAT in a developmental model and find that only ICAT identifies a perturbation-unique cellular response. Taken together, our results demonstrate that ICAT offers a significant improvement in defining cellular responses to perturbation in scRNA-seq data.</p>

<p><strong>Availability and implementation</strong>
https://github.com/BradhamLab/icat.</p>

<h2 id="voltage-gated-sodium-channel-activity-mediates-sea-urchin-larval-skeletal-patterning-through-spatial-regulation-of-wnt5-expression">Voltage-gated sodium channel activity mediates sea urchin larval skeletal patterning through spatial regulation of Wnt5 expression</h2>

<p><a href="https://journals.biologists.com/dev/article-abstract/150/10/dev201460/310744/Voltage-gated-sodium-channel-activity-mediates-sea?redirectedFrom=fulltext">Development, 2023</a></p>

<p><strong>Abstract</strong>:</p>

<p>Defining pattern formation mechanisms during embryonic development is important for understanding the etiology of birth defects and to inform tissue engineering approaches. In this study, we used tricaine, a voltage-gated sodium channel (VGSC) inhibitor, to show that VGSC activity is required for normal skeletal patterning in Lytechinus variegatus sea urchin larvae. We demonstrate that tricaine-mediated patterning defects are rescued by an anesthetic-insensitive version of the VGSC LvScn5a. Expression of this channel is enriched in the ventrolateral ectoderm, where it spatially overlaps with posterolaterally expressed Wnt5. We show that VGSC activity is required to spatially restrict Wnt5 expression to this ectodermal region that is adjacent and instructive to clusters of primary mesenchymal cells that initiate secretion of the larval skeleton as triradiates. Tricaine-mediated Wnt5 spatial expansion correlates with the formation of ectopic PMC clusters and triradiates. These defects are rescued by Wnt5 knockdown, indicating that the spatial expansion of Wnt5 is responsible for the patterning defects induced by VGSC inhibition. These results demonstrate a previously unreported connection between bioelectrical status and the spatial control of patterning cue expression during embryonic pattern formation.</p>

<h2 id="ethanol-exposure-perturbs-sea-urchin-development-and-disrupts-developmental-timing">Ethanol exposure perturbs sea urchin development and disrupts developmental timing</h2>

<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0012160622002123?via%3Dihub">Developmental Biology, 2023</a></p>

<p><strong>Abstract</strong>: Ethanol is a known vertebrate teratogen that causes craniofacial defects as a component of fetal alcohol syndrome (FAS). Our results show that sea urchin embryos treated with ethanol similarly show broad skeletal patterning defects, potentially analogous to the defects associated with FAS. The sea urchin larval skeleton is a simple patterning system that involves only two cell types: the primary mesenchymal cells (PMCs) that secrete the calcium carbonate skeleton and the ectodermal cells that provide migratory, positional, and differentiation cues for the PMCs. Perturbations in RA biosynthesis and Hh signaling pathways are thought to be causal for the FAS phenotype in vertebrates. Surprisingly, our results indicate that these pathways are not functionally relevant for the teratogenic effects of ethanol in developing sea urchins. We found that developmental morphology as well as the expression of some ectodermal and PMC genes was delayed by ethanol exposure. Temporal transcriptome analysis revealed significant impacts of ethanol on signaling and metabolic gene expression, and a disruption in the timing of GRN gene expression that includes both delayed and precocious gene expression throughout the specification network. We conclude that the skeletal patterning perturbations in ethanol-treated embryos likely arise from a loss of temporal synchrony within and between the instructive and responsive tissues.</p>

  </div>
  <div class="colophon">
    <p>
      <span id="pub-date">Published on 12 Jul 2023</span> <span class="separator">&bull;</span>
      <span id="social">Find me on <a href="https://github.com/dakota-hawkins">Github</a>!</span>
    </p>
  </div>
</div>


  </div>
  <script src="/scripts/responsive.js" type="text/javascript"></script>
</body>

</html>